src/views/v2/month/grid-header.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * View: Month View - Grid Header
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/views/v2/month/grid-header.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version TBD
+ *
+ */
+// @todo: Check if we're keeping these template tags or how is that we'll be populating data here.
+$days_of_week = tribe_events_get_days_of_week();
+global $wp_locale;
+?>
+<header class="tribe-events-calendar-month__header" role="rowgroup">
+	<?php foreach ( $days_of_week as $day ) : ?>
+			<div
```

Comment:
The text within should probably be wrapped in an appropriate heading tag and a time tag with datetime attribute (all while still being wrapped in a div as I'm sure that'll be helpful for layout)?

You can nix the `aria-label` if you make the updates above I believe.

Reply:
This is just for the day of week - Like "Monday", "Tuesday" and so on. So I'm not sure if the time tag makes sense. Thoughts?

Reply:
Ahh, you are right on the time tag, good call!